### DISCUSSION TITLE
Upgrade Instructions

### DISCUSSION THREAD

**Discussion by alienyak** [Leantime/leantime#798] [Q&A] [answered]
Are there available upgrade instructions for a system that is already in place? I have mine hosted in a shared web hosting environment and had to customize the .htaccess file in order to make it work.

Thanks in advance,

Daryl

**Comment by marcelfolaron** [comment, marked as answer]
Hi Daryl,

Usually it's just a matter of uploading the entire directory and replacing
the files. Things like configuration and userfiles will remain.
Now the changes to your htaccess file would have to be re-applied. Just
download your htaccess file and replace it once you uploaded all the files.

Sincerely,
Marcel

**Reply by alienyak** [reply]
Thanks for the quick reply.

Daryl
